deps/picotls/lib/openssl.c

```diff
@@ -738,26 +761,14 @@ static void async_sign_ctx_free(ptls_async_job_t *_self)
     free(self);
 }
 
-int async_sign_ctx_get_fd(ptls_async_job_t *_self)
-{
-    struct async_sign_ctx *self = (void *)_self;
-    OSSL_ASYNC_FD fds[1];
-    size_t numfds;
-
-    ASYNC_WAIT_CTX_get_all_fds(self->waitctx, NULL, &numfds);
-    assert(numfds == 1);
-    ASYNC_WAIT_CTX_get_all_fds(self->waitctx, fds, &numfds);
-    return (int)fds[0];
-}
-
 static ptls_async_job_t *async_sign_ctx_new(const ptls_openssl_signature_scheme_t *scheme, EVP_MD_CTX *ctx, size_t siglen)
 {
     struct async_sign_ctx *self;
 
     if ((self = malloc(offsetof(struct async_sign_ctx, sig) + siglen)) == NULL)
         return NULL;
 
-    self->super = (ptls_async_job_t){async_sign_ctx_free, async_sign_ctx_get_fd};
```

Comment:
this is another one -- perhaps better to get https://github.com/h2o/picotls/pull/432 updated and merged, so we can remove most, if not all, picotls changes from here.